jest.config.cjs

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  roots: ['<rootDir>'],
+  preset: 'ts-jest/presets/default-esm', // or other ESM presets
+  testMatch: ['**/?(*.)+(spec|test).+(ts|tsx|js)'],
+  moduleNameMapper: {
+    '\\.(gif|ttf|eot|svg)$': '@jupyterlab/testutils/lib/jest-file-mock.js',
+  },
+  transform: {
+    '^.+\\.svg$': '<rootDir>/jest.svg.transform.js',
```

Comment:
currently hitting problems around svgs - even though we've tried both adding a transform and a module name mapper to prevent them being loaded.

```
 /Users/stevejpurves/dev/thebe-core/node_modules/@jupyterlab/ui-components/style/debug/bad.svg:1
    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
    ^

    SyntaxError: Unexpected token '<'

      at Object.newLoader [as .js] (node_modules/pirates/lib/index.js:141:7)
      at Object.<anonymous> (node_modules/@jupyterlab/ui-components/lib/icon/labicon.js:25:35)
```